client/state/sites/selectors.js

```diff
@@ -73,6 +73,38 @@ export const getSite = createSelector(
 );
 
 /**
+ * Returns a the ID of the primary site.
+ *
+ * @param  {Object}   state  Global state tree
+ * @return {?Number}         The ID of the primary site
+ */
+export function getPrimarySiteId( state ) {
```

Comment:
Makes sense. That would also ensure compatibility with `SitesList.getPrimary()`, so I'm changing it now.